front-end/src/components/Reactionbar/CommentReactionBar.tsx

```diff
@@ -0,0 +1,83 @@
+// Copyright 2019-2020 @paritytech/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import React from 'react';
+import styled from '@xstyled/styled-components';
+
+import { reactions } from './reactions';
+import { useCommentReactionsQuery } from '../../generated/graphql';
+import ReactionButton, { ReactionButtonProps } from './ReactionButton';
+
+interface Reaction {
+	id: number
+	reaction: string
+}
+
+interface Props {
+	className?: string
+	commentId: string
+}
+
+const CommentReactionBar = function ({ className, commentId }: Props) {
+	const { data, refetch } = useCommentReactionsQuery({ variables: { commentId } });
+
+	const reactionMap: { [ key: string ]: ReactionButtonProps; } = {};
+
+	reactions.forEach((reaction) => {
+		const reactionButtonProp: ReactionButtonProps = {
+			count: 0,
+			reaction,
```

Comment:
Can be deleted as well.

Reply:
same as above ReactionButtonProps require a reaction variable

Reply:
same as above, done in https://github.com/paritytech/polkassembly/pull/576/commits/2c632b64dce7eacf0f22b462a17133f8868cde0c